rook/template/values.yaml

```diff
@@ -181,12 +181,18 @@ clusters:
       # Requires Prometheus operator from apps!
       # Enable after apps is installed!
       installPrometheusRules: false
+      installServiceMonitor: false
 
     networkPolicies:
       apiserverPeers: []
       nodePeers: []
 
   workload:
+    monitoring:
+      # Requires Prometheus operator from apps!
+      # Enable after apps is installed!
+      installPrometheusRules: false
```

Comment:
I am not that sure that we need alerts in wc for rook

Reply:
I've only included the option for sc in the template and the autoconfig will ask so one can skip if it is unnecessary.